clickhouse/migrations/schema_10_cache_hits.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS default.cache_hits
+(
+   `request_id` UUID,
+   `organization_id` UUID,
+   `created_at` DateTime64 DEFAULT now(),
+)
+ENGINE = MergeTree
+PRIMARY KEY (request_id)
+ORDER BY (request_id, created_at);
```

Comment:
nice, let's change the order by here to this.
`organization_id, created_at, request_id`

Look up how Clickhouse does order bys, this dramatically change the query's performance

Reply:
I ordered it by your example and other migration orders. https://github.com/Helicone/helicone/pull/1174/commits/eae63d83ca5b50a8070bff828897e5120edbdd1d